lib/foundation/http/dio_logger_interceptor.dart

```diff
@@ -27,12 +26,13 @@ class LoggingInterceptor extends Interceptor {
 
   @override
   void onRequest(RequestOptions options, RequestInterceptorHandler handler) {
-    // Don't log image requests
-    final ext = extension(options.uri.toString());
-    if (_kImageExtensions.contains(ext)) {
-      super.onRequest(options, handler);
-      return;
-    }
+    // NOTE: Commented out while new image request system is still new
```

Comment:
Logging all image requests seems unnecessary, maybe we could just log failed ones selectively

Reply:
I'll fix it myself, need to merge this so another refactoring can be done.